package/kernel/mac80211/patches/701-add-missing-pci-id-for-ubiquiti-branded-qca988x.patch

```diff
@@ -0,0 +1,80 @@
+--- a/drivers/net/wireless/ath/ath10k/core.c
```

Comment:
this patch needs to be inside its own patch. also you need to send the mac80211 stuff upstream to the kernel people

Reply:
Its merged in kernel 4.16+
So if backports are updated this patch could be dropped

Reply:
@blogic Since it's already a patch I guess you meant "commit"?